stories/core/Header.stories.tsx

```diff
@@ -0,0 +1,209 @@
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { Meta, Story } from '@storybook/react/types-6-0';
+import React from 'react';
+import { action } from '@storybook/addon-actions';
+
+import {
+  DropdownMenu,
+  getUserColor,
+  Header,
+  HeaderProps,
+  IconButton,
+  Input,
+  MenuItem,
+  UserIcon,
+} from '../../src/core';
+import HeaderButton from '../../src/core/Header/HeaderButton';
+import HeaderLogo from '../../src/core/Header/HeaderLogo';
+import HeaderBreadcrumbs from '../../src/core/Header/HeaderBreadcrumbs';
+import SvgHelpCircularHollow from '@itwin/itwinui-icons-react/cjs/icons/HelpCircularHollow';
+import SvgVersion from '@itwin/itwinui-icons-react/cjs/icons/Pin';
+import SvgNetwork from '@itwin/itwinui-icons-react/cjs/icons/Network';
+import SvgNotification from '@itwin/itwinui-icons-react/cjs/icons/Notification';
+
+export default {
+  title: 'Core/Header',
+  component: Header,
+  subcomponents: {
+    HeaderBreadcrumbs,
+    HeaderLogo,
+    HeaderButton,
+  },
+  argTypes: {
+    appLogo: { control: { disable: true } },
+    breadcrumbs: { control: { disable: true } },
+    actions: { control: { disable: true } },
+    userIcon: { control: { disable: true } },
+    menuItems: { control: { disable: true } },
+    children: { control: { disable: true } },
+  },
+} as Meta<HeaderProps>;
+
+const buildClickHandler = (menu: string, close: () => void) => (
+  index: number,
+) => {
+  action(`Menu '${menu}', Item #${index} clicked!`)();
+  close();
+};
+
+const buildMenu = (menu: string) => (close: () => void) => [
+  <MenuItem key={1} value={1} onClick={buildClickHandler(menu, close)}>
+    {menu} item #1
+  </MenuItem>,
+  <MenuItem key={2} value={2} onClick={buildClickHandler(menu, close)}>
+    {menu} item #2
+  </MenuItem>,
+  <MenuItem key={3} value={3} onClick={buildClickHandler(menu, close)}>
+    {menu} item #3
+  </MenuItem>,
+];
+
+export const full: Story<HeaderProps> = (args) => {
+  return (
+    <Header
+      appLogo={
+        <HeaderLogo
+          logo={
+            <svg
+              viewBox='0 0 16 16'
+              xmlns='http://www.w3.org/2000/svg'
+              aria-hidden='true'
+            >
+              <path d='m12.6 13.4c-1.2-1.5-2.1-3.1-2.4-5.5-2.7 3.9-4.6 4.2-5.7 2.4l-1.2 5.7h-2.2l3.5-14.1 1.8-.4c-.1.5-1.4 6.2.6 7 2 .7 4.6-8.5 4.6-8.5l2.2.4c-1.6 3.7-1.6 7.6 1.1 10.9l-2.3 2.1' />
+            </svg>
+          }
+          onClick={() => action('Clicked on the title')()}
+        >
+          Microstation
+        </HeaderLogo>
+      }
+      breadcrumbs={
+        <HeaderBreadcrumbs
+          items={[
+            <HeaderButton
+              key='project'
+              menuItems={buildMenu('Project')}
+              name={'Project A (Super Size Edition)'}
```

Comment:
Minor, but no need curlies if it is simple string value..

Reply:
Should probably be handled by prettier or linter or something...